s3.tf

```diff
@@ -24,6 +24,16 @@ resource "aws_s3_bucket_acl" "log_shipping" {
   acl    = "private"
   bucket = aws_s3_bucket.log_shipping[0].id
   count  = length(var.log_shipping_oidc_fully_qualified_subjects) > 0 ? 1 : 0 // Only actually create the ACL if a service account used for log shipping has been specified.
+
+  depends_on = [aws_s3_bucket_ownership_controls.log_shipping_ownership_controls]
```

Comment:
`depends_on` is a code smell, it shouldn't be necessary, it indicates a flaw somewhere else usually.